src/microbe_stage/organelle_components/ExternallyPositionedComponent.cs

```diff
@@ -39,13 +44,15 @@ public virtual void Update(float elapsed)
 
         Vector3 middle = Hex.AxialToCartesian(new Hex(0, 0));
         var delta = middle - organellePos;
```

Comment:
Also, now that I come back to this after a long time, I'm a bit confused by the name of this variable. Perhaps it could be worth renaming it to better reflect what this is.

Reply:
I dont understand what that delta is as well, so any suggestion?

Reply:
relativeOrganelleVector?